canonical-web-and-design/vanilla-framework
# Add alt attribute to images that do not have one

## Done

Fixes https://github.com/canonical-web-and-design/vanilla-squad/issues/877

## QA

- Search for images without alt or with empty alt. I used the following: (<img(?!.*?alt=(['"]).*?\2)[^>]*)(>)
- Verify there are none
- Go through the changes, verify the alt text is plausible
